src/Lean/Elab/Tactic/Omega/MinNatAbs.lean

```diff
@@ -0,0 +1,139 @@
+/-
+Copyright (c) 2023 Lean FRO, LLC. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Scott Morrison
+-/
+
+/-!
+# `List.nonzeroMinimum`, `List.minNatAbs`, `List.maxNatAbs`
+
+`List.minNatAbs` computes the minimum non-zero absolute value of a `List Int`.
+This is not generally useful outside of the implementation of the `omega` tactic,
+so we keep it in the `Std/Tactic/Omega` directory
```

Comment:
Presumably this should no longer refer to `Std/` in this way (I haven't checked any other files for Std references)